core/templates/services/utils.service.ts

```diff
@@ -39,13 +43,22 @@ export class UtilsService {
   // The function here is to check whether the argument is a string. So, we
   // cannot have any specific type defined for the argument and the argument
   // is given a generic type of Object.
+  /**
+   * @param {Object} input - the object to be checked
+   * @return {boolean} - true if input is string, false otherwise
+   */
   isString(input: Object): boolean {
     return (typeof input === 'string' || input instanceof String);
   }
 
   // The function here is to check whether the two arguments are equivalent or
   // not empty or not. So, we cannot have any specific type defined for the
   // arguments and the arguments are given a generic type of Object.
+  /**
+   * @param {Object} a - the object to be checked
+   * @param {Object} b - the object to be checked
```

Comment:
The first object to be compared, the second object to be compared.

Reply:
I have fixed the documentation.